### PR TITLE
Assignments 1 and 2

### PR DESCRIPTION
**Pull request by amcarrero**

### CODE REVIEW

File: Assignment2/amcarrero-AdrianMora.rdf [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <rdf:Description rdf:about="#Class01">
+	    <includes rdf:resource="#Sensor029"/>
+	    <includes rdf:resource="#Computer101"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Sensor029">
+	    <hasMeasurement rdf:resource="#Measurement8401"/>
+    </rdf:Description>
+	<rdf:Description rdf:about="#Measurement8401">
+	    <hasTemperature> 29 </hasTemperature>
+	    <atTime rdf:datatype="http://www.w3.org/2001/XMLSchema#date">2010-06-12T12:00:12</atTime>
```
Comment (ocorcho) [inline Assignment2/amcarrero-AdrianMora.rdf]: This should be dateTime

File: Assignment2/amcarrero-AdrianMora.turtle [hunk 2]
```diff
@@ -0,0 +1,11 @@
+:Class01 vocab:includes :Sensor029 ;
+    vocab:includes :Computer101 .
+
+:Sensor029 vocab:hasMeasurement :Measurement8401 .
+
+:Measurement8401 vocab:hasTemperature "29" ;
```
Comment (ocorcho) [inline Assignment2/amcarrero-AdrianMora.turtle]: Literals should be datatyped. Same for next triple

File: Assignment2/amcarrero-AdrianMora.turtle [hunk 3]
```diff
@@ -0,0 +1,11 @@
+:Class01 vocab:includes :Sensor029 ;
```
Comment (ocorcho) [inline Assignment2/amcarrero-AdrianMora.turtle]: Missing prefixes

File: Assignment2/amcarrero-AdrianMora.rdf (hunk 1 above)
Comment (amcarrero) [reply Assignment2/amcarrero-AdrianMora.rdf]: Just committed some changes, I hope it solves the problems.

File: Assignment2/amcarrero-AdrianMora.rdf [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    <rdf:Description rdf:about="#Class01">
+	    <includes rdf:resource="#Sensor029"/>
+	    <includes rdf:resource="#Computer101"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Sensor029">
+	    <hasMeasurement rdf:resource="#Measurement8401"/>
+    </rdf:Description>
+	<rdf:Description rdf:about="#Measurement8401">
+	    <hasTemperature rdf:datatype="http://www.w3.org/2001/XMLSchema#integer"> 29 </hasTemperature>
+	    <atTime rdf:datatype="https://www.w3.org/TR/xmlschema11-2/#dateTime">2010-06-12T12:00:12</atTime>
```
Comment (ocorcho) [inline Assignment2/amcarrero-AdrianMora.rdf]: Wrong namesapce for dateTime, but I will accept it.

**Pull request merged**